jdbc/src/main/java/org/apache/metamodel/jdbc/JdbcDataContext.java

```diff
@@ -289,6 +289,7 @@ private boolean usesCatalogsAsSchemas(DatabaseMetaData metaData) {
         } catch (SQLException e) {
             throw JdbcUtils.wrapException(e, "retrieve schema and catalog metadata", JdbcActionType.METADATA);
         } finally {
+            FileHelper.safeClose(rs);
             close(null);
```

Comment:
Hmm I think we should remove this line. Seems like it's not needed since `close(null)` will just do nothing. Maybe it's a left-over line from some previous version of the file where the result set got closed properly or something :-)